CONTRIBUTING.md

```diff
@@ -19,7 +19,11 @@ with the help of [pre-commit](https://pre-commit.com) and
 [clang-format](https://clang.llvm.org/docs/ClangFormat.html) so that
 our source code has a consistent coding style at all times.
 
-You can install pre-commit as follows.
+First, you have to install pre-commit package as follows.
```

Comment:
I tested it again in Ubuntu 18.04, which uses `Python 3.6.9`. It works fine when I downgraded clang-format from `v14.0.6` to `v13.0.0`. However, clang-format `v13.0.0` produces some urgly style so I decided to use more recent one.

Since clang-format `v13.0.1`, it shows some errors as follows.
```
root@ef6d1a0d3deb:~/uftrace# pre-commit run
[INFO] Installing environment for https://github.com/pre-commit/mirrors-clang-format.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/root/.cache/pre-commit/repoc4ou7q8x/py_env-python3/bin/python', '-mpip', 'install', '.')
return code: 1
expected return code: 0
stdout:
    Processing /root/.cache/pre-commit/repoc4ou7q8x
      Preparing metadata (setup.py): started
      Preparing metadata (setup.py): finished with status 'error'

stderr:
      ERROR: Command errored out with exit status 1:
       command: /root/.cache/pre-commit/repoc4ou7q8x/py_env-python3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/root/.cache/pre-commit/repoc4ou7q8x/setup.py'"'"'; __file__='"'"'/root/.cache/pre-commit/repoc4ou7q8x/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-90_24veu
           cwd: /root/.cache/pre-commit/repoc4ou7q8x/
      Complete output (6 lines):
      Traceback (most recent call last):
        File "<string>", line 1, in <module>
        File "/root/.cache/pre-commit/repoc4ou7q8x/setup.py", line 1
          from __future__ import annotations
          ^
      SyntaxError: future feature annotations is not defined
      ----------------------------------------
    WARNING: Discarding file:///root/.cache/pre-commit/repoc4ou7q8x. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
    ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

Check the log at /root/.cache/pre-commit/pre-commit.log
```
I have searched the error message `SyntaxError: future feature annotations is not defined` and many posts explains that it requires python3.7 or higher.

If we want to support older python versions, then clang-format version has to be downgraded otherwise, python3.7 is required.

Reply:
It means Ubuntu 18.04 users will see this issue because its default python version is `3.6.9`.

Reply:
OK. @kangtegong Please update the doc as @namhyung mentioned.

Reply:
Just amended the commit, thanks! :)